nth-prime.json

```diff
@@ -0,0 +1,34 @@
+{
+  "#": [
+    "The tests that expect 'false' should be implemented to raise",
+    "an error, or indicate a failure. Implement this in a way that",
+    "makes sense for your language."
+  ],
+  "cases": [
+    {
+       "description": "first",
+       "input": "1",
+       "expected": "2"
+    },
+    {
+       "description": "second",
+       "input": "2",
+       "expected": "3"
+    },
+    {
+       "description": "sixth prime",
+       "input": "6",
+       "expected": "13"
+    },
+    {
+       "description": "big prime",
+       "input": "10_001",
+       "expected": "104_743"
+    },
+    {
+       "description": "weird case",
```

Comment:
A more descriptive description would be better.
maybe: "there is no zeroth prime" or "uses 1 based indexing"